src/plugins/chart_expressions/common/color_categories.ts

```diff
@@ -15,27 +15,38 @@ import { isMultiFieldKey } from '@kbn/data-plugin/common';
  */
 export function getColorCategories(
   rows: DatatableRow[],
-  accessor?: string
+  accessor?: string,
+  isTransposed?: boolean,
+  exclude?: any[]
```

Comment:
can you clarify what's the usage of this `exclude` thing here?
I've seen it is used only once with `null` value... Is there any other use case for it?

Reply:
Yeah it's used twice, whenever I'm getting categories for the datatable. Previously it was not possible, AFAIK, to have actual `null` values in the data. But with the datatable we have this, currently the numeric color assignments ignore the `null` rows altogether and apply no styles. We opened  https://github.com/elastic/kibana/issues/190174 to discuss this.

But for now we need a way to exclude `null` values from the color assignments thus this `exclude` option.